src/testpackage/Test2.java

```diff
@@ -0,0 +1,15 @@
+package testpackage;
+
+public class Test2 {
+
+	public static void main(String[] args) {
+		String hungry = "yes";
+		
+		if (hungry == "yes") {
+			System.out.println("eat something....");
+		}else {
```

Comment:
Hi ramlearn,

Your code looks good. please add one more check

Reply:
Hi,

I have updated the checks, pls check